src/components/Table/styles.tsx

```diff
@@ -54,8 +54,8 @@ export interface GridStyleDef {
 function memoizedTableStyles() {
   const cache: Record<string, GridStyle> = {};
   return (props?: GridStyleDef) => {
-    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight } = props || {};
-    const key = `${inlineEditing}|${grouped}|${rowHeight}`;
+    const { inlineEditing = false, grouped = false, rowHeight = "flexible", cellHighlight = false } = props || {};
+    const key = `${inlineEditing}|${grouped}|${rowHeight}|${cellHighlight}`;
```

Comment:
@bdow wdyt about a `JSON.stringify(props)` so that new props going forward are handled for free/

Reply:
I like the idea. I wonder how that'd look... Will try.

Reply:
Would be worried props may not always be returned in the same order, so could be tricky. Might be able to do something like `Object.keys`, sort, get values, then stringify.

Reply:
Pushed up a change, let me know what you think! Thanks for the suggestion.